config/example/dbconfig.txt

```diff
@@ -9,7 +9,7 @@
 ## This value must be set to the version of the paradise schema in use.
 ## If this value does not match, the SQL database will not be loaded and an error will be generated.
 ## Roundstart will be delayed.
-DB_VERSION 12
+DB_VERSION 13
```

Comment:
Should use SS13 DB version 13 on something epic

Reply:
Removed DB version incrementing.